src/test/java/POM/pages/BasePage.java

```diff
@@ -0,0 +1,44 @@
+package POM.pages;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.interactions.Actions;
+import org.openqa.selenium.support.PageFactory;
+import org.openqa.selenium.support.ui.WebDriverWait;
+import java.time.Duration;
+public class BasePage {
+    WebDriver driver;
+    WebDriverWait wait;
+    Actions actions;
```

Comment:
Please, make the variables protected, because protected access gives the subclass a chance to use the helper method or variable, while preventing a nonrelated class from trying to use it.

For example

     ```
    protected WebDriver driver;
    protected WebDriverWait wait;
    protected Actions actions;
    ```

Reply:
Thank you for the feedback @Berichicko 👍🏽

Reply:
**I hope you get the job QA AUTOMATION JAVA , or something else soon!**